package.json

```diff
@@ -8,16 +8,21 @@
     "babel-plugin-transform-es2015-modules-commonjs": "^6.26.0",
     "babel-preset-env": "^1.6.0",
     "core-js": "^2.5.1",
+    "coveralls": "^3.0.0",
     "escodegen": "^1.9.0",
     "eslint": "^4.10.0",
     "eslint-plugin-mozilla": "^0.4.9",
     "eslint-plugin-no-unsanitized": "^2.0.1",
     "fancy-log": "^1.3.2",
     "gulp": "^3.9.1",
+    "gulp-inject": "^4.3.0",
+    "gulp-istanbul": "^1.1.2",
     "gulp-rename": "^1.2.2",
     "gulp-replace": "^0.6.1",
     "gulp-transform": "^3.0.5",
-    "gulp-zip": "^4.1.0",
+    "gulp-util": "^3.0.8",
+    "gulp-zip": "^4.0.0",
```

Comment:
Why did you downgrade the gulp-zip dependency?

Reply:
I'll change it back to the same